package-dev/Runtime/SentryInitialization.cs

```diff
@@ -39,6 +46,38 @@ public static void Init()
                 SentryNativeAndroid.Configure(options, sentryUnityInfo);
 #elif SENTRY_NATIVE_WINDOWS
                 SentryNative.Configure(options);
+
+                // On Standalone, we disable cache dir in case multiple app instances run over the same path.
+                // Note: we cannot use a named Mutex, because Unit doesn't support it. Instead, we create a file with `FileShare.None`.
+                // https://forum.unity.com/threads/unsupported-internal-call-for-il2cpp-mutex-createmutex_internal-named-mutexes-are-not-supported.387334/
+                if (options.CacheDirectoryPath != null)
+                {
+                    try
+                    {
+                        _globalMutex = new FileStream(Path.Combine(options.CacheDirectoryPath, "sentry-unity.lock"), FileMode.OpenOrCreate,
+                                FileAccess.ReadWrite, FileShare.None);
+
+                        Application.quitting += () =>
+                        {
+                            try
+                            {
+                                _globalMutex.Close();
```

Comment:
In the event of a crash (or any reason this event doesn't fire) will it ever recover?

Reply:
added a comment: "We don't really need to close, Windows would release the lock anyway, but let's be nice."

Reply:
I've tested that manually